packetizer/class_extractor.py

```diff
@@ -1,20 +1,38 @@
 import ast
+from dataclasses import dataclass
+
+
+@dataclass
+class NameDto:
```

Comment:
python acronyms are recommended to be capitalised.
```suggestion
class NameDTO:
```